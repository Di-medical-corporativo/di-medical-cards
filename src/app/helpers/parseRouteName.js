export default (routeName) => {
    const splittedName = routeName.split('-')
    const parsedName = []
    splittedName.forEach(e => parsedName.push(e.normalize('NFD').replace(/([\u0300-\u036f]|[^0-9a-zA-Z])/g, '')))
    return parsedName.join(' ')
}