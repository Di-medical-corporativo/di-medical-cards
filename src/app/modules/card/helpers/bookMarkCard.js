function bookMark () {
  if (window.sidebar && window.sidebar.addPanel) {
    window.sidebar.addPanel(document.title, window.location.href, '')
  } else if (window.external && ('AddFavorite' in window.external)) {
    window.external.AddFavorite(location.href, document.title)
  } else if (window.opera && window.print) {
    this.title = document.title
  } else {
    alert('Press ' + (navigator.userAgent.toLowerCase().indexOf('mac') !== -1 ? 'Command/Cmd' : 'CTRL') + ' + D to bookmark this page.')
  }
  localStorage.setItem('bookmark', true)
}

export default bookMark
