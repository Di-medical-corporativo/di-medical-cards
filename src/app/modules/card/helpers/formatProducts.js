export const formatResult = (result = {}) => {
  const productsList = []
  for (const id of Object.keys(result)) {
    productsList.push({
      id,
      ...result[id]
    })
  }

  return productsList
}
