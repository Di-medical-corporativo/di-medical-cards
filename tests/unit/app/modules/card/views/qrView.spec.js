import { shallowMount } from '@vue/test-utils'
import Qrview from '../../../../../../src/app/modules/card/views/QRCodeView.vue'
import { surBase64, corpBase64 } from '../../../../../../src/app/modules/card/helpers/base64Logos'

jest.mock('vue-router')

describe('QRCodeView', () => {
  test('should match snapshot', () => {
    require('vue-router').useRoute.mockReturnValueOnce({ params: { sucursal: 'sur', id: 1 } })
    const wrapper = shallowMount(Qrview)
    expect(wrapper.html()).toMatchSnapshot()
  })

  test('should set current link depending on route params', () => {
    require('vue-router').useRoute.mockReturnValueOnce({ params: { sucursal: 'sur', id: 1 } })
    const wrapper = shallowMount(Qrview)
    const link = 'https://card.dimedicalcorporativo.mx/sur/1'
    expect(wrapper.vm.currentLink).toBe(link)
  })

  test('should set dotsColor depending on sucursal route params, (Sur)', () => {
    require('vue-router').useRoute.mockReturnValueOnce({ params: { sucursal: 'sur', id: 1 } })
    const wrapper = shallowMount(Qrview)
    expect(wrapper.vm.dotsColor).toBe('#218d9b')
  })

  test('should set dotsColor depending on sucursal route params, (Sur)', () => {
    require('vue-router').useRoute.mockReturnValueOnce({ params: { sucursal: 'corporativo', id: 1 } })
    const wrapper = shallowMount(Qrview)
    expect(wrapper.vm.dotsColor).toBe('#c28400')
  })

  test('should set cornersColor depending on sucursal route params, (sur)', () => {
    require('vue-router').useRoute.mockReturnValueOnce({ params: { sucursal: 'sur', id: 1 } })
    const wrapper = shallowMount(Qrview)
    expect(wrapper.vm.cornersCOlor).toBe('#187876')
  })

  test('should set cornersColor depending on sucursal route params, (corp)', () => {
    require('vue-router').useRoute.mockReturnValueOnce({ params: { sucursal: 'corporativo', id: 1 } })
    const wrapper = shallowMount(Qrview)
    expect(wrapper.vm.cornersCOlor).toBe('#684700')
  })

  test('should return base 64 image depending on route params, (SUR)', () => {
    require('vue-router').useRoute.mockReturnValueOnce({ params: { sucursal: 'sur', id: 1 } })
    const wrapper = shallowMount(Qrview)
    expect(wrapper.vm.base64Image).toBe(surBase64)
  })

  test('should return base 64 image depending on route params, (CORPORATIVO)', () => {
    require('vue-router').useRoute.mockReturnValueOnce({ params: { sucursal: 'corporativo', id: 1 } })
    const wrapper = shallowMount(Qrview)
    expect(wrapper.vm.base64Image).toBe(corpBase64)
  })

  test.todo('Test download image')
})
