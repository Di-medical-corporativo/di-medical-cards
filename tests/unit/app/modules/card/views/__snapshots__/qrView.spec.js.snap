// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QRCodeView should match snapshot 1`] = `
<div class="card-qr">
  <div class="card-qr__image">
    <q-rcode-stub width="500" imgclass="img-qr" myclass="" downloadbutton="" buttonname="Download3" height="500" image="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAfQAAAH0CAMAAAD8CC+4AAAC91BMVEUAAAAViYkRh4cViIgTiIgQhoYmjZcdjZUQhoYOhYUaiZAYhIsVgYkViYgTh4cSiIcWg4kTgocTiIgRh4cWiYkXhYoOhYURhocNhIQRhYYZhIgZhosSh4gahowUiIgPhoUUg4kaiYwYhosQhIYWhooVhYkfnKkMhIMTiIgViYkQhoYRh4cThocWhIcXhooTiIcUiIgWiIoXho0XhooXhoohnaoSh4cSh4gWh4kIeYYWhYkTiIgSh4cWhIkTiIcUiYkTiIgViIoaiY4XhYoTh4kVhIl9u9gfnKoUiIgQhYYRhocXg4gTg4cZiYsWh4oaiYsTh4kRhogYiIsPhYUXiIodio0ZipAThYcUhYkSgYUUg4d6utd3uNYOhYURhIYUhYkWh4oYgYUUgocVg4ginaoUhYcXiIsWhokTgoYWhYkMg4QejZASiId4udcWiYkUg4YTg4YlnatytdMbmqgRhoYUiIgPhYYViYoWh4tytdJ0ttV3uNYQhYV5udcSg4Z0ttN0tNJ8uth5udYfnKkLg4J3uNYWiYsbiowYiIsSgoUTgoUhnKoUhIgjnKpwtNEJg4IKg4IRh4YhnaoRhoYWioohnKsbi44knaolnasPhYUZmad4uNcLgoQhnasOg4QknasVhYd0ttVzttNustEUiYgXioojnqxzt9Ykn6wmn60UiIp0t9UinKp2uNZ5udcPhYd2uNUinKpwttRts9AmnqsXioojnqwPhIUPgoUnn6wejZMTgYRzttQUhooimqkkm6lzttMlnKolnawhnaocm6ginqx0t9Uknqx0t9UjmqginKkjnatss9MjnqwhnKt1t9UWh4kim6gknqx2t9Qln6xyt9UZiosZi4x1t9Ybio5xtdIlnakkmqhxttVxttUgnasln60ajItxt9UfnKoknqxzt9V1t9Qmn6xwttMnoK4noa4im6tbrsgnoK1nstJqstNzt9UajIwqoa4roa0gna1rsdFutdMenKsXioojnqt/vNgXi4kWiYn7daw4AAAA+HRSTlMA/fj8+/cDBfj1CBwa/vr6Dhf6+uoL79L0uUIRrEb86jo+FKmCRfvyy7rz55c9H+zZk0YtI/zvtlI/KuXBMPzVx45EM6Yl/fjez72FTYiFcLGdf8adi31raFZE9+/jf19XQTYn/ol4Yls4zoj28uF2cAsH9drQtKJzFuvf3teDSAz69fDnwZeBZ2VHk4UWEO/s4eDVwqWGYhH37+e8urCMe3AjHfPu7ebQq6qdfu3Ko5VrWjAr+MKrjoSAaE9KQTQrJRvn59q6j4xbSi/dtJ6EfXI8N+fOr6emf2Q5H+LY1MrKsK56amFWPZxzUEfZ2Mp5/OvfxpSFyb5ZlAMAACDPSURBVHja7MGBAAAAAICg/akXqQIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAYPbtXjWBKAjD8Km8tL0J9xZs7HaXxXVhW29gK2NhIS4S2xSpxCJgYRMCSRGIbao0XxVBTPyNdWbe5xaGGb4zwwEAAAAAAAAAAAAAAAAAAAAAAPj3RgHufBUB3nwOA7x5e3oM8KX1GlfdAFeKtN1ZBriSpJHK+wBPklRSvA5wZJRqq74L8GNXdOVJgBu7okvNIMCLfdHVsJlzI0mlnYqqe1Gk2quY8E50U/1oSHNOjPUr587qQ6YD2TzAgUaH6l6AfZWOxC8B5j3rWLnk0mreUieiBSHeumGkUzlxzriHvs7UHNhtm5c6V07YyVo2iLXFiPcl00XTj1aAVZUu67+T4s2a6ZqM9ZxVm0jXdGZcW21al7oup9lNKlJJNLszlSSa3ZlVpD91JsR4c3qlbqg3vNmNKca6pV3xn9mYhW6brtjGm/LN3r3HthTFcQD/teu2KtuYdcyYdyMzJijiFYIOQTwiHu0E2wTRsBUxtbDNBPWWGYbZJDPZPCbiEfOYeYv3JAsjIvFI/CORyDln/2jXlXa77e65vXKv5nz+W7Lmpvn2d+45v55zW2wgPFirgAkct02ED+N9NsYHkCzCj+kCm8cHjB8GwlNhMdtBFyBKzhK+DFns1h4g8gh/OXlsf0VA+GIkFEwPWT8+AGyxEhoG1o8PBM8NhIr5AuvV/PdOFRJKWeyJVP+9C4SWiT1UVi6KQgSv2mgZ69h8Th5+FgkudXrn2OPnZOG4pQIEySwk9Kzsxi4Hz5KOvgRBirMJL+whBrJzLUlvEZZ6xjkiwFl21lF6BTqMBY7wr02EF/boCrl5ZQ8d6x6ECO3Q0DN/AUZau5OwQ75G6ABPz8RGeKmV4kaXdwO9kkIixFk2m5PYZex04AnQ22wmQhSylZu0bLjJ0UdArziHCGFlXRpJFWAXi00D1B7Sps4eFy8Dt/BfZZeAVki5kQhRx87ASEiD3eRWCkhdUK0bPwMjnTLsRvf0DnXqAkZ49otPErNhD3cLgFaxiQiQxW7r0nmkxx50x6uBUlUhEaAcGKlU63AzuRUaoFOSZyTUzKxHI51a3MK9x0An48dGNsD/T/JxS/qnl4DO7fs5hJKB/VKvZApSMYck226gczErm9AxsbMvUqnXY065FelAZcuP0wZCJY+1aKRyD3tR81IDVDIvnCU0jOxbVqk80GNvaio1tLd2M6GQxU4yS+RmEvau5mU6UKl6YyS8ZbMTEBLR5GJfaioOA42Mz9ZswpeVnXGTyBnsW25RPdDY85D3qt3ASl0iX1NxK5Ly3wGN25/MrNTlLT0Xt0r38VEoUKjiuWrPZluiJXIG81F6jWaUzyjmt2rPYmt1aXzXYV4sTx9TRJRZbiKty2Hfu0hDcxfzVXOtGnj7do5HQ/4+MJKwYf4slwvSKZZvpDWmEuAppaenFSNjwTvNijFyu3PE9kiBv0LGrNCAKNoNGAj0nugwhdTc/Meh4o3xz4GftMgGlSooSKWKDA9XKpVqtTJCmbz+xVLgNC1ZrZwyEuRk6bC4I1fBJWWlYmz/aSCCBasUwdcF7ZSjoy+1vQvhO8YbiU/WDOBjJOKkUE+e3QFaGLMX2a3VgowkIISCteCkXYvs4seA/6YihFYCvQI9pqU/8OAW/Txe+BaaJcirYWntoZltYcgubAHISG+EUMd2rvKMaFAoFCgB/DdFgVAy0Eu/iwXQHSjiVe8lvns15fxDD4tzlrcTcmnYNAI8zYx0/FvYapCR3gqEIl2hb1DY34VKcUKM0FXCQociLExqre0mj9wv+urHF27hHfroRX0cFjVKmxUTFd6UfOcEDbgbP9YRetw0kBFH6ApX6DcUjR/eNJFCHwUC1CdhofSl+V8P+zOhy6kCHroguxnQzJBlnYKdxT+6F7hp2z8MNXQ+Jqv5uyN05Ao99JgaIVVyW5FC7w9CPMN+SLUcf1kNvm3+4LXY6yhD93Sjnwo5rGsHblJO7Nua0B7kxKPSof3VrftOpIAYoQcJDf2SHvtHd8D2OB182FNuppi/c4c+HTiEzO/uTL0XyJordB6oJ3L9RW3A0y3gj1fWg3dVVgPhYv5GETqXlE7IrmFqKMiZI/S4fxB6J+GhP9FjMejLKqrBm8w6zmI3PPczdIidiuwU40HOHKGr5BU65GOR6D5WHgYv3r4nHPL8DR20o5Fd8GCQsa7/MPQoEKbagqnR77q4zXUEamOmv6FD7CBkt1YD8iXH0MGGRaQv87LrIoNjn7S5yu/QYUg0slsG8iXL0HeXYlHVetlJe/E0j5s6deiwGtlFyarb7qlr3D8KPciP0KFSj8WOXcM5xL/JJp7yRAg9xDHAN8wHJ23X6MQZ0IpF3eOvaCH0RsKURGX02ivXY6GJZsGsTsMi4g/2Xq4FTkPnnZ/SXalMnprWNxQ4he5MO7YpPjh+U0zawpA/ld6xndvFo3trOV4VY39V9METqye0eIM7E/olqiJHxZyc5Bm6yp/QoQyLrbYAOOxp3ow/vcX/0GG5o+M1Dpy2h9v/SAGfUtQIqbcvnhysQI3Ch6VpnQv/Ua72rjpxNUeoO2OUyiDUKCg8akYHaKHXjOTOKuSk6ry/T6+m0JV/Qp/UEaE2O5q9aldysOvKQepBa9p7XPQgcl1TOXGgeKHfsmDRlb3iqspfG4k70zd+oc8FX6IQQhEDodEhZNcNfFroiCWxDXIzaCRA7HB1A/ojYk5b8PSbvTsNaqOM4zj+fzbZXJCEJCWEgkBRuU9BQG4EbItYQVAKHtRWi9WKeFF0HB1Hrdaraqu2jkfH26rjfY863uM5juN43zrexzheP2Z94W6STXaTbFjSxGkdP9OZtps+yXS+PLtPdsNSNW2Bkr1hm5nUlpfboZR3+IZAdEso+j4eMMsR6lHV6lHG8icUeyUGhfz+2NFTfd1F/0r+Eor29W3qGw8lI3otADZBfscxAHtRXLtxCAt2XlCXc2rEtsl6UurPlx8LZbBkdZNCxiO+yKeFNTcjIrobsFysGmVFEIM86vocecXCI8AOv+JxknXsYHTbp7MpcGusffxZH++nWMndm4zoe7oBDMvR557pe1kQwNDReupDzN9oyWZ/KIOhRtxkhKQmh8JGrMEhaztaW2seYgZ/Wu84hbhKAqkdjpah1pIFzA2JoeRzA+AJRzeqo7uWBkZxjsuH+koa7MHwQ4HXLvwVftWdrQuC1S9MVnR6evFsChx13UcU5QHlFZjzkhE9kwEo0R99z0B0oeEEFxHZjqlBkCDc7N+Uecwp/hD3UMgGH0Ts5opKM4lsCytutkPkO4GCgnuK4qENaSSpn8riIDECyFNF94Sj5yyFxBgcZVrTVd0OkWFd4Ekgas/2LwnXPMIgOrwwWdHpq71nU+HWlynKjYpF/GsnJSE6TQKonnf0Y3tDK36GgE3ypgz/Jr6SghaWQcTdk04habt7gL/hC8510zT8SUaVV4SqEcTSQtHz5Ogao3K61krRgzkHwBYcE/rKkx4R1svRDTsa3XTdbEosfuNEinRT+Dtcb7sxGdE3S0Fs84suHF5FMnMf/MoVm1ohyqWAwsshKttAKiM8wJCfTpIT3BBFrP56Nwmxojv2p6ATOP80Tyelts52wRJ8adsFMzkU8oRTALw5yYpOl1wxmxovPBij+tmh+4EnI/o7AByu+UV3HENhVTdDxJSbetdKmTPCn22EdZQiHOJjsHveJ1G9F4BwbQapubIQoIrulKKHRtmjRtHymp5CzQ9D2seTFp2evmY2Nc75Qbv6YR8kI/q7UsQ03dF3k6KfYSKFdQKAy1WbtjOA2zfQpgwQnCeQWuayyyByv0uiQYjOcFGk3nyt6KpR8WUuXL5t2/JKG1G/AGA4edHp5aLZ1Fh8V/Rx/ZPAGv74P5M10/Nc+lfvPMAGSWncKK/bQi7gAcOBJKkVpFlMasdcLkBSvJqICssA8BdStAmLdnSXNMp6IcXXfc9DAjM6Ob5hsDmDAeibcyHXbSO9HjtqNjX2jv65IQ8E3rkd+l6yjukO2/yin6l+FSn60ep/ZQXY+yQytQBwF5DSms6t8CtbZpL2x9L4aYrBtBQilh4r+nKnPEpbZgVngMz6aAOApcHobs3oC0szSa/vZlPl4S0U4azb9pOiv5ja1bt29CNIqaAdwMWktK8U/XwSdVsADJFC+jof/Lbm1pNkwA5Y+imWDVYAeTGjD0AepSm9xAiFYhaOnuXQjJ6Da3NIJ9Mds6ly+yUU4YPT9xOj/5KE6BnucJSrdJ6cYXuoo9sBLNOIvpwD2DYKsa3Oh5/l1DEKaAHQYKNYXFJ0Y8zo8ihthR2Q2I2AxQ6JIjqnGZ1O9RzbSzqZX59NlReepQh/3C1Gf+2kJJyR8wBsOBydBaNPDJRmKzVuMyUUvcsIYIxkTTXGQInwKfIMaUsrxXaKAHhiRc+wyKM0TUPEynKX77nnyLQPfkxH9PWcofxC3dVDcz311U968VAx+qVJOff+t2EiGN0Qil5pRQTjBZrRhTjRe6Sk8sSpK7XCb21tDsl6pVddR7FlA+DSY7xP77UCWEdxjEud2zubyW9lll1v9CYr4DvCTPqY3pBPzaW++k2fHHrYazft8KVVc4vUMy0c3RCIfqCASLXa0aOP6bwcPdcNIFCtsNYNP76xmcLqGGBYT7Hl2jWi1/kAtp60mU8BwLIzKShnCCJ27NzRc4oBWK6t0r2GL5pNkU+3kNr9H9/22gMU35FzX0/Pkw/p6uhNHkRwrg5E5/RGNwRmugNAvTQjRoKXTj01/aS0CnFmeiPTiL7KCmCYtPVvBdBSSCFV+RDJ0Xnt6NQJSf446fTyObMp8vuJpHbnaz9/u6PRzR0A2jdERzcNNvC8VcJLv/Oc9/qcUHTMHZ2To9dyACqJxrKcAkQs6hMWLohKKbYsAJz63HsgussCIJu0rbcDzilSWKY3+kHwK75e72R/+orZFPnSRCqn/fLzjzsa/QQGoDqTVMf0ILOCSQo13+gHkWiGB9hU/QAkArjhdIp0C8QAJoolswyAJxydhaJTNYByeZTGdLW4SKHZCEDH7p3aGAIaRkykyyWvp+jAftRFpHa/vujL5vg0LNtGpF69a9OOfqY6ukOOfqRVSuqFn7W1m6JtZoBljeaaCsaY0fsYwK8hTS0AWkilTGd0mkQQ19lN+tyXol384vtIxfSFruj7k5bCTQD+lj/3Lp+cSW50WwNkzslRUrK9/24/id53AsKw5iEdeTGjH+RRjYq5/6ghlXK90bsYZMUVhaTLMw+nZrIvfotUHtAV/eL43+FivZBU0fdKSnSjHJ16BAQ8tC2TVHb3YOMKaU3GAULZKoq2yAoRixm92QoIxc06zjTKzGvV0W8hLQUOhFXPmEmXq9+eTYXnniWVLTsSPcff3P4ozS+6Q1d0Qyj6Iif8eqpIbcoH4B0SleJvoNVMkTKyoB2dsgF5VCybBcDSTAqVFkV0Z9yrbJNQEGqaSJctN6RkH/+widQSP6av6oAk25za6FJSEX8BRVjjBcAODk1oewVFMD/C4kWv9MUcRZXvrDYR0cGIvAw0yMLRO51AmZm0HAwVrnMh6fLsd4uTH73osXlG1/7+9HMhWVpIKYnOpOiK1SJa6kil+3AA8idn1kHEV5hIyba7BVrRlaPU6QoHrfBJX0qL2gF420gmLSnD0bd7gK1jpKXODRVh63S3zuwv3Zr06tc8lZToT3Z6IMlKp/lGd+uNzoLR6UAHRA9NkMLEuRDC8z9nCUTO7HoKqxvyQCu6alSfctSKy51ghkkSlQACLl9FQd3lyjNyMxyAIRtpKUEk30Ab6XLifbcn+zL7FVvmGV04IMY9Z7bCr7WQkhTdHic6DUPiKFlhIz9bU6vP32uQguqqIWLermYKaKsoA6KjQxmd6vLVo2xNQz5IqRtJtNAqAEL+BpP/oSmvAIAZg9HTygBw17tIw4gBan8D1oGVpM+DN/yW3OpvzDM6Nh90vuggvz2OHs4utzABEq4iGGFeb9kcOqLLx/Qgcw/8rA2l65ctW5/dIOUALD1mkq3x9xM8xZt6Dt7j6N1P8XkgOaODqS6tSqPC0akgH5K84pp3D15W25hvgV91s+KNl7Wl54ADtld7/M05Lhidah2Awdlwzx4TmRRDjhcx+BoXkd7uXz2cxMP74h9It4UQMQawEIS0NFGE44z6orPo6HkR0d2A8U2SmbqMCGIGhgBOtcpqW4IAZnS7DZAInqX17zjALOHoTI6uGqV+XnZGG/mZcxkApnjIzfFy9IwOAYwzcDw/QLEMIya+s4n0Mn/4/R0/SeWPSsIOPmMeM12L0HB0DoXp/xDFblK+uaOzYHRZ0wKo8ZvGSKWwh2NQMhRXZFKPE8yqmunc/nFHMec9LgoyD1uhsFH8auKWUlBVDQMzGhjbaKYYVnoQm6NmxEb6mR985fuX7vjs1qKjkns6VluzgJgsk+e7KNoMRN0UV3c7IEyRUh0Ao3pTNwA2QgoZB9xshEzgT3nCTJEWtvJOuaCd82UXENEedqOhODP0tHbAMkUqY61OY3CUACf3SAEpHNMSfEYB1k1PdjCW10cyV2M7g+hcE8XSCU3lB6TTfJ346jePSfM+Udc8TXodDYObMWZwOxxGo8HPuaD0/DUUU9pkuzPbRHGZsj32M9JIpZQ5IjaZ+uyGJemkYpvIXcDAmIHfVLEnxdR29FCZg+ctFm/rEc0kqWox+B5VvDjjpFdS6z66ZK3BYeEt+dNTkQ/alvetZczovnn6GDMth6VslML6G9caLdYzKaYVdgGavNsXUSJsT1/93e17J3aVlXRbVak2x/3eV859v3dT20qbzk3RCtsqC+oyKA5b78rKlfVmkmUWrNJ4WvWogsqVVabYz9hcULDKTJKqAhepZNYV1FNspiUC4vCcujyTEnPiW49dN/9Td0VX0/9S7kCGuIT89W2UKPNbj322eJ5ruRPpf6lma8Fc+JILMilhW765Yz5n7/a+i/6XcqvtmBPLX7cvJc781Hf6uz+/hf6Xapkt0MO5ZNmqHen+ypU6D/BFN9D/Um7GDn18JTMuStyJVz9cpOtt2yX0v1SzTUK3sr4LMihxD7703H9/qo8eXjZoIh1yri3OqqN4Vj5a0uBsKOlqo6Qbd0M/dlnjBYWUMNvVP/3Xj+qbANZNOiy3MkMFaaufdroZRG7r9b2UbCXQTxB/eUtH0ilhT11XNMdU37UX8OUCDGOkwyE88+xOmsaqw7cps+ePUpIt4jFf1s436yhRr96xOP5U35lv3jx3dMC9p77o4LSjL7wZgGB0M45BtLGJkuweJMA5WbuXOdGDe9zsRffRLqwcYPqic8jTjJ62QABYde3owhXDXoAhv4qSKy0fCeG80wf2JjjbX4+zk/+UdmHlAPRFd4DlkoYeAMbgx1+qOu1gWEdJNtWOBAmWJcNNOZSAtz7TzF70Fu269Ed3A1rR63kA02YKyNxsZ/ickszUCQEJM1pLup7MoHn75gqty+9X0q5rge7oRu3oq41Ag4tkGaWMn6Bkq/MicQIA5iupbSqk+bHddc1R/7kTNEmJPsDAKijM1NRGyfcmdpy7uGbdSLeZ5uGZ2O/fir6nXdYC3Qs5g3b0DsAySqlmbkVyOPNbu1bUk24vP3fUf2spl5ToSwDnGKVcfT6ShnHVfbXjbWbS45IrT46OfvKrtKu6RXd0BuyufccC4yil3iiP5GJrO7af39Rrorn88PzeUZfVd6ET8Ka6FTMHrZ6otGlHzxnbsG3byFgGKb0fJ/oAg72CYjKPv7+IlKpmZtIpIG1qJIcy+qdOGG8mfSogIOkMvHeodrxqjske/ab9ip3qJyDH0VzbYuE5h9ti9U43haKrK/XxHG/heG5t34SNQh6PE301A/JdFMseBrA1qk+8GVlW8I+THuu1j3o53mq1dD5JethKkBqCwF8+OJpBcXx/TeT+/UPaFRQOFjOEWJcuio7ef8pWhFjO2KArer0bEKbNFEOpHTiQwtIFgMsI/NFusFs2cpzRbYB9Y4WJdKhqEZAynLdvpJA0Pf2COvpRL9EuYNEZDIAgCKHsR5gWAIZwdFtFMUIEANa+qjmjyzc8aHTFjm5QRk9joehpboOR53mh3eF/pUF9/wcvUqq4cYWZNGz5Ur2Lf4F2fk2XBUp7OzZ/3uIzQLR1UBU9o88OkYOf/Hxzh5eHpHw3HdHTyiHKX7/iyG5XVHSmip4HsGB0g5vjOQd/amcDB2awNJEe406klru8q5403HWNav/+IO3snhSbC0LZYKWJRFWrl+QB4Jl88zKRORsiftMT6f6/jfVYIfI2KaPHn4AOzmFpWbcozu49zQ0gGB0Ojndeu4rIdgQD5xwgXfZgSDFhY+4aiu2Vc5TnZ76inVzvuQDah5pJZjt4I/wM+1JALQNw2ZSJZN1ZdgDeNjl6LmnaNx8hxdm92sd0I8LRGTMem0mSEbCtw6TPMFLP17eIYnpVeae662gntxmAu8em3t+rou9ZDODcI0kho5QB2KwjOtWXOhBy7mi86EyODvDyP5yQfvaHPqbt+BdsbeymWJ5V3Ongmp382x5W8IBwvTliow8I35tyKYCNyubSftcHgMtURNe2Zv2p5260MkiKN8jRER09TxG9iubPNoB/g3XQRTGceF3oPM3iV2inVgLg8qi5VGsPR+/nAe4EUuqvsRogoMGmjB5fTmVXtb96v3Z0dzi6xUU7b3Xkz5gomvnKol3jpFydVHQiOtEt4ejbAZxiprDmRh8EoP3wJvk0bC7pUdjDAyh3aUbnFDPdRYmwbce/gpV0x73J+F+0M5sxQqi2UZQutxzd/A97d/fbUhzHcfz7Ozun7Sl9WrXrzOM8tGaqjVEZqmWLkplhMpawZWKrZPEQhgtxtXgWhAjiwgU3blx5SISESNy64lKEf4CrT3JcOKd61qOtx7VNH36vyy3tzXvn/H47v3N+597PD7c3hf3Q+MNN9LfRdXEbgK1FjE7mXoaSaOk2U64L6WN9Q1kP6h0s/078S+x69HaLArGT0tT9wWdBJUfaDats8+gv9QAYnVqU6LrNdigohUAn5XrzYza38AWVsRjg6KNcnon9pp9Lht08+8ckqMRQF9H/RJ/yCGCHihqdzl5BKShK3v3mLyxPDepl/Z96CLCvpDzu6dEHJKCBUtqeRKFisw+a6f+i03FA6ckb3V2o6DR9N0pD6rXmGddTx/onKmNHAUfe6Pf16CNS+g1KnlOj0AhbpxL9b/RkHXC0yNEpGECJxKbkVv+UWl6lMhYA6s5RLrOgj+krJeAqkSm5qj6VPDZE9P/RhwB4ix2drK2ygpK4up6yHX6szeTK+fbIHgBhyrVG1qOvlwB/81AMmuiqJNFkoq8B0FLEMV03uBil4e2ibJc+qGsu5TyTm8+AEOUaFgAxFd06im/YrkDjPeWhyUXvF4Glv4pumeTFGaNgJAoFJdBylrK9fL38SDk/3rQkClj6KZspBMA+c+IF1wwq35NOY6Xe3rZUdPFfot9lQECPfoIypkQLFV13draCUpCHKduL20cuUPkyHQAQMFGWPgsAcYH+olyNfW0/GXgaFBYy/SF68vhg7k3RCOvX+eJZ72poMU1EF1w0aVM7ZJSCPJ+yfT5S1gttwzBuFmzcvVm/icKzGAC7n7WD5FbtyGzWvuDXDzuM2CAlySgpAu7zpLoBYC1l7HEDDelYevRJmxsQUQKWbsr26SuVMU8CgKWPjDpPAsjcI9cnAGLMSUaHRAYWI9VZEbj26wXu8TbKaF8KIGQi1XR3av1U51J/I7ZSoY50nWkggRKQt1GWm1/L+kH1BbICyHGr4Se7oWFQo+ujOrNv3EsTzPscrJ5d2U+q/TZA6qJ8utxgONpJuvVXAcg/Spu0GrOX0Q+eRgC+oUx0u4sKw7rlKkPR3TqRM6y/onJ2VwBgCw2ksw899EPFRHEi+tQDYDY80m8Rs+7aaFGY6D+YmRUs7ad8elBnYSsOOUnTvKcFgE0f/wdk7XNJE6nWb4e2qk8Fja6zng0xBUUmD1BliVug8o1Hwt2bO476RagaHgkC06NT+zSo6qSxrd3zw8fGfVD59xFlJnpyY/dOJ2VrDkGQ6qQVkfiNcKPXAdV2p+ESAYN0tHV+fLUfqt1BKvDpXWfqivhQZN51VFmG/UiZ5XYzaKQ7rlXuemEB6drWMqSwWQwKVFf6KM0UAURmF6QZlGPKmMgEBsxSP6axbW8mXVMAYAyiw6ZANT6HVIaLMwXVtvmAqKCItKGqsswNMWTMurLHTIlovdRPEzzhOhj4V6+hCc5NbigAVlEuz2mGjMutHspoijiQpignh8gY3eGiAjNN790toIgSLqos5p1jcrqONN4aJKIZqPN1kkH7Vm+6O/NHzpOR+eK9egbbaspn/YyW9Of8jUvoZyMJB4OKXXnaZPi6FcBsMxWeeW68wYKiuW6mStPe19rTGOnYc95KmuD7yEj2UdvV3bGoccbTXS7K5nq2tfHhMsrPlQz3bIq0ngvmyXD+4rzGSOtAMxmd7+nppyKZkuxtkOsZiiFMXNlqmn7ieGCFRRBZYeP7Bokrb1PmDA6HHywKhFataJFlWbJIkmSf3N/B7E7iKofZ6nQF966Zs2DlyJYz4feRWOJevUPEvwpU3rDOGTk756qnghmBpZIgMvwdW5y4qtC8f7D7wViLJOLPRucSVz3MywZ3LFosMfyWknASV2XaBsOx0aiCXztNXBXyrLux2isw5OfjJ/hq5ew6nfDlP+Ib+Ay+ii3bFvDZkU1hb4mrZm3DY7lTuxVB4qrbsvgqmcFI6SCu2plXRmQYSXOIq37tp731yIgRVwtc8xcz6Nx8ua1GON9msq+tlD2Buclq2uzFD8IIcbUi2CEhJcGv0NSQdSFo7PxQryXW+ChUa4mrJXMTAISVxNUS57so6q4TV1sOXo66h4irLV3jtuPE1Zj9u780E1dj1t8/QVyt2f+RuJqzstIeXuYKYCpxHMdxHMdxHMdxHMdxHMdxHMdx3Hf24EAAAAAAAMj/tRFUVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVVaU9OCABAAAAEPT/dTsCFQAAAAAAeAr2ruSm0+/p5wAAAABJRU5ErkJggg==" qroptions="[object Object]" imageoptions="[object Object]" dotsoptions="[object Object]" backgroundoptions="[object Object]" cornerssquareoptions="[object Object]" cornersdotoptions="[object Object]" fileext="png" download="false" downloadoptions="[object Object]" id="qrImageBase64" value="https://card.dimedicalcorporativo.mx/sur/1"></q-rcode-stub>
  </div><button class="card-qr__download" style="background-color: rgb(33, 141, 155);"> Descargar </button>
</div>
`;
